include/ssl_key_handler.hpp

```diff
@@ -102,9 +102,11 @@ inline bool validateCertificate(const std::string &filePath,
         unsigned int retErrCode = X509_STORE_CTX_get_error(storeCtx);
         X509_STORE_CTX_free(storeCtx);
         X509_LOOKUP_free(x509Lookup);
-        if (isTrustChainError(retErrCode))
+        if (isTrustChainError(retErrCode) ||
+           (retErrCode == X509_V_ERR_CERT_NOT_YET_VALID))
```

Comment:
Are we going to push this upsteam ?
https://github.com/openbmc/bmcweb/blob/c94d37490b0090a2a4a0b6bad51cd8ab1339051e/include/ssl_key_handler.hpp#L82

Reply:
Master is fixed with a different solution